data/aerial-photography/index.html

```diff
@@ -13,12 +13,13 @@
 title: Aerial Photography Products for Utah
 categories: []
 ---
-<p>All aerial photography downloads available from AGRC, with the exception of the Google imagery, have a UTM NAD83 zone 12 north meters projection.</p>
-<p>Common problems and issues people encouter with the imagery products are <a href="{{ '/data/aerial-photography/imagery-problems/' | prepend: site.baseurl }}">compiled into a document</a> with descriptions and workarounds. <strong>If you are having issues, please look for your issue there first</strong>. {% include contact.html subject=page.title contact=site.data.contacts.imagery %}</p>
+<p>All aerial photography downloads available from AGRC, with the exception of Google Imagery, have a UTM NAD83 zone 12 north meters projection.</p>
+<p>If you have any problems or issues with the Google Imagery products, take a look at our <a href="{{ '/data/aerial-photography/imagery-problems/' | prepend: site.baseurl }}">Imagery Format Problem Fixes and Workarounds
```

Comment:
This is where I may have messed up the formatting, but it was a little hard to tell with all the html stuff. I'll check the preview.